packages/x-components/src/x-modules/search-box/components/__tests__/search-input.spec.ts

```diff
@@ -28,7 +28,7 @@ function mountNewSearchInput(overrideProps: Partial<SearchInputProps> = {}): Tes
     localVue,
     propsData: overrideProps
   });
-  const input = wrapper.vm.$refs.input as HTMLInputElement;
+  const input = wrapper.vm.$refs.inputElement as HTMLInputElement;
```

Comment:
Use native ways to find components and elements instead of `refs`.
`wrapper.find(getDataTestSelector('search-input')).element as HTMLInputElement`